src/components/scenes/TransactionsExportScene.tsx

```diff
@@ -145,23 +145,25 @@ class TransactionsExportSceneComponent extends React.PureComponent<Props, State>
     this.setState({ endDate: date })
   }
 
-  handleAndroidToggle = () => {
-    this.setState(state => ({
-      isExportCsv: !state.isExportCsv,
-      isExportQbo: !state.isExportQbo,
-      isExportBitwave: !state.isExportBitwave
-    }))
-  }
-
   handleQboToggle = () => {
+    if (Platform.OS === 'android' && !this.state.isExportQbo) {
+      this.setState({ isExportCsv: false, isExportBitwave: false })
+    }
+
     this.setState(state => ({ isExportQbo: !state.isExportQbo }))
```

Comment:
If you press the radio button once on Android, it will select the option. If the press the ratio button again, it will de-select the option and leave nothing chosen. Then, when you hit submit, you will get a crash, because `files[0]` is undefined.

The way I wrote the handler in my original review comment would have avoided this possibility, by ensuring exactly one item can be selected on Android.

Reply:
You can't submit if nothing is selected. The Next button is hidden. This is also true on iOS. I did test this case. But it's fair to say the user shouldn't be allowed to select nothing.